debugging/book-library/script.js

```diff
@@ -28,19 +27,22 @@ const check = document.getElementById("check");
 //check the right input from forms and if its ok -> add the new book (object in array)
 //via Book function and start render function
 function submit() {
-  if (
-    title.value == null ||
-    title.value == "" ||
-    pages.value == null ||
-    pages.value == ""
-  ) {
+  if (!title.value.trim() || !pages.value.trim()) { // simplify validation
```

Comment:
I like the simplification. There is still one more expression missing from this check. Please, think which one.